dlt/sources/helpers/rest_client/paginators.py

```diff
@@ -136,6 +146,7 @@ def __init__(
         self.total_path = jsonpath.compile_path(total_path) if total_path else None
         self.error_message_items = error_message_items
         self.stop_after_empty_page = stop_after_empty_page
+        self.has_more_path = has_more_path
```

Comment:
Please either compile jsonpath (as with `total_path`) for parity/perf, or add a note explaining why not.